distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1384,6 +1384,8 @@ Monitor system logs with the *LogViewer*, a convenient "viewing portal" for inco
 
 The LogViewer displays the most recent 500 log messages by default, but will grow to a maximum of 5000 messages.
 
+The top left button labeled POLLING/PAUSED will toggle live retrieval of incoming logs from the backend.
+
 Log events can be filtered by:
 
 * Log level (`ERROR`, `WARNING`, etc).
```

Comment:
I moved the note above the warning so it immediately comes after the log level filtering blurb. I think it's best to keep it inside the note tag to draw extra attention to it since it's a point of concern that has been brought up a couple times before.